communitter-frontend/src/utilities/CommentService.js

```diff
@@ -15,11 +15,52 @@ export class CommentService {
             });
 
             if (!commentResponse.ok) {
-                throw new Error("Failed to create comment");
+                throw new Error("Failed to create comment: ", commentResponse);
             }
             return await commentResponse.json(); 
         } catch (error) {
             console.log(error.message);
         }
     }
+
```

Comment:
why did you create extra service for comment http requests? We have a fetchWithOps function which takes the token from the local storage and set the baseline for http requests.

Reply:
Instead of creating method for every call made from frontend, I thought It would be unmanagable to change code within each component. I decided to create a service where I apply error handling as well as the API calls.